### DISCUSSION TITLE
essay/2023/08/07/developer-is-a-creator

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#118] [Comments]
# essay/2023/08/07/developer-is-a-creator

개발자는 어쩌면 크리에이터라 불려야 하지 않을까?

https://jeho.page/essay/2023/08/07/developer-is-a-creator.html

**Comment by likegravity** [comment]
그렇다고 볼수도 있겠네요 재미있게 읽었습니다

**Comment by thenewseason** [comment]
안녕하세요.
사이드 프로젝트로 앱을 운영하고 있는 개발자로, 
재호님 블로그와 건물주 브랜치를 즐겨보고 있습니다.
원하시던 건물주 해방?은 축하드립니다 ^^

혹시 앱 개발하실 때 디자인은 어떻게 하셨나요?
혼자 계속 만들다보니 UI 나 디자인 면에서 한계가 느껴져서요..
다른 후발주자 앱들 보다보면 역시 업체에서 기획자를 두고 만든건 다르구나 생각하면서
좋은 점은 따라하고 있습니다. ^^;

주변을 돌아봤을 때 같은 개발자라도 디자인 센스가 있는 사람도 있던데요,
재호님도 센스가 있으셨는지 아님 어떤 다른 방안이 있었는지 궁금합니다.

**Reply by BenjaminKim** [reply]
개발자 중에 디자인 금손들도 분명히 있는 것 같습니다.
저는 완전히 똥손인데요. [커피한잔 랜딩 페이지](https://withcoffee.app/)를 보면 알 수 있습니다.
다행히도 아내가 금손이라.. 필요한 디자인들이 있으면 아내에게 만들어달라고 부탁하고 있습니다.

**Reply by thenewseason** [reply]
커피한잔 실사용자는 아니지만 가입해서 이것저것 보긴했었거든요.
컬러나 팝업 등의 작은 애니메이션이 잘 쓰여서 차분한 느낌이 좋았습니다.
앱에 있는 컬러도 아내분이 선정해주셨나요?
커피콩 등의 디자인도 직접해주신거고요?

**Reply by BenjaminKim** [reply]
커피한잔 앱의 디자인은 예전에 카카오에서 함께 일한 훌륭한 디자이너가 만들어주셨습니다.
(아마 이제는 앱을 안 쓰고 계시겠지만... 혹시 아직까지 앱을 이용 중이시라면 탈퇴를 부탁드립니다.
정말 필요해서 사용하는 사람들끼리만 소개시켜주고 싶습니다.)

**Reply by thenewseason** [reply]
답변 감사드립니다.
앱은 탈퇴하였습니다. 
앞으로도 많은 활동 기대하겠습니다.
감사합니다~!

**Reply by BenjaminKim** [reply]
고맙습니다. 저도 응원할께요!
